### PR TITLE
Updates in preparation for the MP 4.0-RC1 release

### PR DESCRIPTION
**Pull request by ederks85**
Signed-off-by: Edwin Derks <ederks85@gmail.com>

### CODE REVIEW

**Review by xstefank — APPROVED** [review]
Looks good.

**Comment by Emily-Jiang** [comment]
@ederks85 It does not make sense to raise a 4.0-RC1 while still contains MP Config 2.0-M1. I suggest to pull in all of the latest releases from other individual specs. I think 4.0-RC1 needs to depend on MP Config 2.0-RC1, which we plan to release this week. I think it is best for other specs to comment which release version they would like 2.0-RC1 to pull in.

**Comment by ederks85** [comment]
@Emily-Jiang you are correct, we need at lease a MP-Config 2.0-RC1 before this PR can be merged.

I had noticed this when creating the PR but didn't mention it because I assumed this would be understood since the changes I made were just the things that could already be done.

**Comment by ederks85** [comment]
And I see now that I have accidentally not set this PR up as draft PR. I have fixed this, and now we should be good :)

File: spec/src/main/asciidoc/required-apis.asciidoc
Comment (xstefank) [inline spec/src/main/asciidoc/required-apis.asciidoc:115]: we have now RC5 available -> https://search.maven.org/artifact/org.eclipse.microprofile.health/microprofile-health-api/3.0-RC5/jar

Comment (ederks85) [reply spec/src/main/asciidoc/required-apis.asciidoc]: Ah yes I see, I have updated the RC versions of all specifications for good measure.

File: spec/src/main/asciidoc/architecture.asciidoc
Comment (Emily-Jiang) [inline spec/src/main/asciidoc/architecture.asciidoc:48]: https://download.eclipse.org/microprofile/microprofile-health-3.0-RC5/microprofile-health-spec.html#_incompatible_changes 
[Health]

File: spec/src/main/asciidoc/architecture.asciidoc
Comment (Emily-Jiang) [inline spec/src/main/asciidoc/architecture.asciidoc:50]: https://download.eclipse.org/microprofile/microprofile-open-api-2.0-RC3/microprofile-openapi-spec.html#_incompatible_changes[Open API]

Comment (ederks85) [reply spec/src/main/asciidoc/architecture.asciidoc]: Updated this link in the document

File: spec/src/main/asciidoc/architecture.asciidoc
Comment (ederks85) [reply spec/src/main/asciidoc/architecture.asciidoc]: Updated this link in the document

File: README.adoc
Comment (kwsutter) [inline README.adoc:109]: You changed microprofile.io to MicroProfile.io on lines 105 and 107, but not here.  Since the URL is not case sensitive, I'm okay either way.  But, we should be consistent.

File: README.adoc
Comment (kwsutter) [inline README.adoc:124]: Should we talk about the MicroProfile working group and its associated mailing list?  Not sure it's necessary since the individual Jakarta EE projects do not talk about the Jakarta EE Working Groups and its mailing lists.  But, since MP is a different type of community, I'm just asking the question.

File: README.adoc
Comment (kwsutter) [inline README.adoc:127]: Is it the most complete?  How is this "contributors" web site kept up to date?  Is it automated?  How does one get listed on this page?  The other list referenced in line 128 is only the committers (not all contributors).  That list below is automated.

File: attic/Processes/eclipse.adoc
Comment (kwsutter) [inline attic/Processes/eclipse.adoc:3]: This reference, we shouldn't change.  In 2016, we were accepted and known as Eclipse MicroProfile.  The Eclipse prefix was a requirement.  Maybe we need another line about becoming a Working Group as of such-and-such date and now we are known as MicroProfile.

File: pom.xml
Comment (kwsutter) [inline pom.xml:39]: Should we put a `<post>` link for the microprofile-wg email list?

File: spec/src/main/asciidoc/architecture.asciidoc
Comment (kwsutter) [inline spec/src/main/asciidoc/architecture.asciidoc:37]: We have to figure out an alternate way of keeping track of our "releases".  The Mx and RCx are not "releases" in the normal sense of the word.  Nobody should be using these Mx or RCx releases in production usage.  The term "release" seems to indicate that they could be.  

I think we are now past the point of counting our releases.  We should just indicate the "next", and then maybe qualify the release with the words "milestone" or "release candidate" to make it clear what the intent is.

File: spec/src/main/asciidoc/license-alv2.asciidoc
Comment (kwsutter) [inline spec/src/main/asciidoc/license-alv2.asciidoc:32]: Should we be changing Copyrights?  Maybe the original should be left as-is since through 2019, it was Eclipse MicroProfile.  And, then just add a new line for 2020-2020 MicroProfile?  Or, the follow the convention in the other files in this project, "..2020-2020 Contributors to the Eclipse Foundation".

File: spec/src/main/asciidoc/required-apis.asciidoc
Comment (kwsutter) [inline spec/src/main/asciidoc/required-apis.asciidoc:35]: Since this is the Platform RCx release, I would think this document should reference the real expected releases of the components (and not reference the RCx releases.

File: spec/src/main/asciidoc/required-apis.asciidoc
Comment (kwsutter) [inline spec/src/main/asciidoc/required-apis.asciidoc:101]: Specific references like this to the github repo tags might have to be with the RC tag, since the real release doesn't exist yet...  Just depends on how accurate one wants to be with the documentation.

File: README.adoc
Comment (aeiras) [reply README.adoc:127]: Currently, Summers is working on fixing with the MP Projects' page which will lead to updating the Contributors page as well. Using git the updates will continue to be automatic by activity, yet there are many broken items which require Summer's help. See MP thread https://groups.google.com/forum/#!topic/microprofile/7xEB4PO9Zqs

File: attic/Processes/eclipse.adoc
Comment (aeiras) [reply attic/Processes/eclipse.adoc:3]: It needs to change to MP alone. The year is 2017 formally, not 2016. :)

File: spec/src/main/asciidoc/license-alv2.asciidoc
Comment (aeiras) [reply spec/src/main/asciidoc/license-alv2.asciidoc:32]: It needs to be MicroProfile Contributors

Comment (aeiras) [reply spec/src/main/asciidoc/license-alv2.asciidoc:32]: This greatly matters because when we are ready to send the MicroProfile trademark, it is going to be MicroProfile across without the Eclipse. 

Therefore, MicroProfile nicely reverts to its original name under the WG umbrella, a huge perk. Questions, please bring them to tmw's after the Steering call is completed, there is the time for a community conversation to clear up any issues.

File: README.adoc
Comment (ederks85) [reply README.adoc:109]: fixed

File: README.adoc
Comment (jclingan) [reply README.adoc:124]: Good point, but I'd like to target next RC or before final to address. I'd like to finalize RC1 so we can push it out and get other release members up to speed on the mechanics of delivery a release.

File: README.adoc
Comment (ederks85) [reply README.adoc:127]: Fixed

File: attic/Processes/eclipse.adoc
Comment (ederks85) [reply attic/Processes/eclipse.adoc:3]: Fixed

File: pom.xml
Comment (ederks85) [reply pom.xml:39]: We added

File: spec/src/main/asciidoc/required-apis.asciidoc
Comment (ederks85) [reply spec/src/main/asciidoc/required-apis.asciidoc:35]: Fixed

File: spec/src/main/asciidoc/required-apis.asciidoc
Comment (ederks85) [reply spec/src/main/asciidoc/required-apis.asciidoc:101]: Fixed

**Review by emecas — APPROVED** [review]

**Review by jclingan — APPROVED** [review]
Looks good based comments. we can tighten things up before RC2 or final.

**Pull request merged**
